### ISSUE TITLE
Popping noise when playing games on Mac OS X

### ISSUE THREAD

**Issue by drakecaiman** [Yabause/yabause#370]
When starting a game, Yabause produces a constant popping sound (a pop every fourth of a second) while playing.

Tried with 0.9.14 and latest dev version (49134be)
Produced regardless of "Enable Higher Sound (Slower)" setting

Console output for starting game:
```
default	07:31:13.054306 -0600	Yabause	<private> marked as needing update structural regions, even though it has received update structural regions 2 times this cycle. Future marking as needing update structural regions might be ignored. <private>
default	07:31:13.275380 -0600	Yabause	Connection to sharingd became invalid
default	07:31:13.457500 -0600	Yabause	errors encountered while discovering extensions
```

MP4 video of popping sound:
[yabause_clicking.zip](https://github.com/Yabause/yabause/files/606493/yabause_clicking.zip)

macOS Sierra 10.12.1
Yabause 49134be (Downloaded 11/22/2016)

Thanks!

**Comment by SuperdorkMightyX** [comment]
Yeah the Saturn has the same popping noise when you play it this is perfect emulation right down to the snapping of the five or six processors you need a power plant or two cords plugged in or one of those big three prong cords with all the power that thing sapps
